Source/WebCore/rendering/RenderScrollbarPart.h

```diff
@@ -66,7 +66,7 @@ class RenderScrollbarPart final : public RenderBlock {
     void computeScrollbarWidth();
     void computeScrollbarHeight();
     
-    RenderScrollbar* m_scrollbar;
+    RefPtr<RenderScrollbar> m_scrollbar;
```

Comment:
Hi, Abigail :).

Since `RenderScrollbar` owns `RenderScrollbarPart` (in `RenderScrollbar::m_parts`), this will create a reference cycle. We probably want to use WeakPtr<RenderScrollbar> instead.

Reply:
I'm actually making that change right now, Wenson checked me on that! (I think this is the first time I've run into a need for WeakPtr organically in the wild?)

Reply:
It looks like the base for RenderScrollbar (Widget) supports `SingleThreadWeakPtr` so that's what I've added here as regular `WeakPtr` doesn't seem to be compatible